frontend/src/Pages/TeamDebateRoom.tsx

```
@@ -0,0 +1,1557 @@
+            if (isFromMyTeam) {
+              // This role selection is from my team
+              setLocalRole(data.role as DebateRole);
+            } else {
+              // This role selection is from opponent team
+              setPeerRole(data.role as DebateRole);
+            }
+          }
+          break;
+        case "countdownStart":
+          // Backend is starting countdown - show it to all users
+          const countdownValue = (data as any).countdown || 3;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)